<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Secmainwindow</class>
 <widget class="QDialog" name="Secmainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>812</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>812</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <widget class="QLabel" name="secpage_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>812</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/mainpage.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="go_summary">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="go_adding">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="title">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>325</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 0px</string>
    </property>
    <property name="text">
     <string>Eat some Food</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="info">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>345</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 8pt &quot;맑은 고딕&quot;;
color: rgb(165, 165, 165);
background-color: rgb(255, 255, 255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>Expense | Meal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="plus_minus">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>325</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>+2000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_2">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>467</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;맑은 고딕&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border: 0px</string>
    </property>
    <property name="text">
     <string>Eat some Food</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>16</width>
      <height>812</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="sum">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>345</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 8pt &quot;맑은 고딕&quot;;
color: rgb(165, 165, 165);
background-color: rgb(255, 255, 255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>Expense | Meal</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
